src/main/java/org/wildfly/security/auth/client/AuthenticationContextConfigurationClient.java

```diff
@@ -148,6 +149,7 @@ public AuthenticationConfiguration getAuthenticationConfigurationNoOverrides(URI
         return configuration;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Would not be appropriate to set this method as Deprecated too instead?

Reply:
This is a private method so probably doesn't need flagging as deprecated.